UefiPayloadPkg/UefiPayloadEntry/UefiPayloadEntry.c

```diff
@@ -517,6 +517,8 @@ BuildGenericHob (
 
   // The UEFI payload FV
   BuildMemoryAllocationHob (PcdGet32 (PcdPayloadFdMemBase), PcdGet32 (PcdPayloadFdMemSize), EfiBootServicesData);
+  // The UEFI payload config FV
+  BuildMemoryAllocationHob (PcdGet32 (PcdPayloadFdMemBase) - SIZE_64KB, SIZE_64KB, EfiRuntimeServicesData);

```

Comment:
why build runtime memory hob here? For other information, the PayloadEntry should already got the information and build a new HOB list to DXE core. Even it is really needed, maybe you need add LINUXBOOT_PAYLOAD flag for this code.

Reply:
fix per mail discussion.